tensorflow_io/bigquery/processing_census_example.ipynb

```diff
@@ -0,0 +1,557 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "processing census using BigQuery TensorFlow connector.ipynb",
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "view-in-github"
+      },
+      "source": [
+        "\u003ca href=\"https://colab.research.google.com/github/tensorflow/io/blob/master/tensorflow_io/bigquery/processing_census_example.ipynb\" target=\"_parent\"\u003e\u003cimg src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/\u003e\u003c/a\u003e\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "upgCc3gXybsA",
+        "colab_type": "text"
+      },
+      "source": [
+        "Install required Packages, and restart runtime."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Tu01THzWcE-J",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!pip install google-cloud-bigquery-storage\n",
+        "!pip install fastavro\n",
+        "!pip install tensorflow-io==0.9.0"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yZmI7l_GykcW",
+        "colab_type": "text"
+      },
+      "source": [
+        "Authenticate."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "tUo3GJNxxZbc",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "from google.colab import auth\n",
+        "auth.authenticate_user()\n",
+        "print('Authenticated')"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "czF1KlC6y8fB",
+        "colab_type": "text"
+      },
+      "source": [
+        "TODO: Set your PROJECT_ID."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "fbQR-bT_xgba",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "PROJECT_ID = \"<YOUR PROJECT_ID>\" #@param {type:\"string\"}\n",
```

Comment:
Usually I encourage people to save notebooks without optputs. But since we can't run this and generate the outputs for tensorflow.org, it might be nice to save this with outputs.

Reply:
I removed all comments.

Reply:
@MarkDaoust template link in https://github.com/tensorflow/io/blob/master/docs/tutorials/README.md is broken.
I'm not sure if you want to link to tensorflow template, or create a separate copy for IO